packages/data/src/plugins/actions.js

```diff
@@ -199,3 +200,20 @@ export function formatErrors( response ) {
 
 	return response;
 }
+
+const formatErrorMessage = ( pluginErrors, actionType = 'install' ) => {
+	return sprintf(
+		/* translators: %(actionType): install or activate (the plugin). %(pluginName): a plugin slug (e.g. woocommerce-services). %(error): a single error message or in plural a comma separated error message list.*/
+		_n(
+			'Could not %(actionType)s %(pluginName)s plugin, %(error)s',
```

Comment:
This tries to provide a more helpful failure message for plugin installation failures.